front/package.json

```diff
@@ -1,16 +1,26 @@
 {
   "name": "coaching-extia",
   "version": "0.0.0",
+  "engines": {
```

Comment:
Remove all code related to back

Reply:
ok je retiens, par contre c'était aussi pour tester le déploiement donc c'était pas intentionnel